docs/specs/src/_hwp_D_rate_limiting_spec.md

```diff
@@ -118,11 +118,6 @@ Note: This is only true of the special case of cap entries as it is invalid for
 
 Many Agents can be created unlike the other Class A entries.
 
-The rate limiting here is implied because all Agent keys are referencing entries in the `deepkey` happ.
```

Comment:
Whether these changes should be kept is up for discussion.

Reply:
no, I think it makes sense to remove it -- if deepkey isn't a conductor service anymore, then it can't have an influence on the rate of agent key changes in other cells, I believe?

Reply:
True.